windowsforms/TaskDialogDemo/vb/TaskDialogDemo.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
We don't include SLN files unless they are absolutely required. We can safely delete this file.